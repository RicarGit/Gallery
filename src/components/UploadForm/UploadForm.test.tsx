import { render } from '@testing-library/react'
import { UploadForm } from '.'

describe('PhotoCard', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <UploadForm photos={[]} setPhotos={() => ''} uploading={false} setUploading={() => true} />
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})