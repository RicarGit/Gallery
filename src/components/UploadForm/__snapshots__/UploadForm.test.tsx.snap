// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoCard should match snapshot 1`] = `
<form
  class="styles__Form-sc-wiyzfq-0 iCRkSN"
  method="POST"
>
  <input
    name="image"
    type="file"
  />
  <input
    type="submit"
    value="Enviar"
  />
</form>
`;
